npm/ng-packs/packages/oauth/src/lib/strategies/auth-flow-strategy.ts

```diff
@@ -80,6 +81,31 @@ export abstract class AuthFlowStrategy {
       .catch(this.catchError);
   }
 
+  protected navigateToPreviousUrl(): void {
+    const { responseType } = this.oAuthConfig;
+    if (responseType === 'code') {
+      this.oAuthService.events
+        .pipe(
+          filter(event => event.type === 'token_received' && !!this.oAuthService.state),
+          map(() => {
+            const redirect_uri = decodeURIComponent(this.oAuthService.state);
+
+            if (redirect_uri && redirect_uri !== '/') {
+              return redirect_uri;
+            }
+            return '/';
+          }),
+          switchMap(redirectUri =>
+            this.configState.getOne$('currentUser').pipe(
+              filter(user => Boolean(user?.isAuthenticated)),
```

Comment:
Boolean(user?.isAuthenticated) is ok but you can use  !!user?.isAuthenticated
double bang (!) . it. just a remider . it is ok